### PR TITLE
Wrap multi-reddit names in the search bar

### PR DESCRIPTION
**Pull request by paulgeringer**
Fixes an annoying visual discrepancy. Not too sure if there are other places where this should go. It appears to work in Chrome on OS X.

### CODE REVIEW

**Comment by bsimpson63** [comment]
Can you include a screenshot of before/after?

**Comment by paulgeringer** [comment]
Before:
Imgur

After:
Imgur

**Comment by bsimpson63** [comment]
This should probably use a more specific selector--maybe `#searchexpando label` will work. Also what do you think about doing "text-overflow: ellipsis;" instead of wrapping?

**Comment by paulgeringer** [comment]
Works for me! Done.

File: r2/r2/public/static/css/reddit.less [hunk 1]
```diff
@@ -2688,6 +2688,10 @@ textarea.gray { color: gray; }
     border-radius:3px;
 }
 
+#searchexpando label {
+    text-overflow: ellipsis;
```
Comment (bsimpson63) [inline r2/r2/public/static/css/reddit.less]: This doesn't seem to be sufficient.

```
display: block;
overflow: hidden;
text-overflow: ellipsis;
white-space: nowrap;
```
seems to get the behavior you want

**Comment by bsimpson63** [comment]
:nail_care:

**Comment by bsimpson63** [comment]
Closing. Comment here if you want to reopen this.

**Pull request closed without merging**